regtests/Dockerfile

```diff
@@ -54,4 +54,4 @@ USER root
 RUN chmod -R go+rwx /home/spark/regtests
 USER spark
 
-CMD ["./run.sh"]
+ENTRYPOINT ["./run.sh"]
```

Comment:
Change to this PR?

Reply:
added when I was debugging this test failure- https://github.com/apache/polaris/pull/493/commits/e5367d96273bf0560f0ce5bef8b003c1e20ffeeb . TBH, I'm not sure how tests in main were passing